COMMITTERS.adoc

```diff
@@ -0,0 +1,182 @@
+= Information for Quarkus Committers
+
+This document contains useful information for Quarkus committers.
+
+[NOTE]
+====
+It is a work in progress, so if you think information is missing/unclear/inadequate,
+please reach out to us on the `quarkus-dev` mailing-list.
+====
+
+This document has a lot of information.
+It is not a set of strict rules, it is here to guide you.
+If you forget to do something, miss something, don't worry, it happens.
+
+Do your best and that will already be a lot :).
+
+== Merging Pull Requests
```

Comment:
One subject I did not see covered is dependency updates. I suppose there is a dedicated process specialization. And maybe a warning that key dependencies like Netty or vert.x should be updated early in the cycle.

Reply:
Yes, I will add something about it.